sim:
  gui: True
  sleep: True
  scene: pile

urdf_root: ../urdf_zoo/models/blocks
object_count: 5
metric: physics
